### PR TITLE
Added Additional Information About Wish Lists Managing

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added Additional Information About Wish Lists Managing:

- Move items to a different list
- Copy items to a different list
- Delete Created Wish Lists

This pull request (PR) ...

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- ...

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- ...

## Affected documentation pages

https://docs.magento.com/user-guide/customers/account-dashboard-my-wish-list.html

- ...
- ...

### CODE REVIEW

File: src/customers/account-dashboard-my-wish-list.md [hunk 1]
```diff
@@ -110,3 +110,35 @@ In the storefront, a customer can create a wish list from their account dashboar
 
     ![]({% link images/images/account-dashboard-wishlist-sharing-email.png %}){: .zoom}
     _Shared Wish List Email_
+
+{:.ee-only}
+## Move items to a different list
+
+1. Select the checkbox of each item to be moved.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:117]:
```suggestion
1. Customer selects the checkbox of each item to be moved.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 2]
```diff
@@ -110,3 +110,35 @@ In the storefront, a customer can create a wish list from their account dashboar
...
+
+1. Click **Move Selected to Wish List** and do one of the following:
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:119]:
```suggestion
1. Clicks **Move Selected to Wish List** and does one of the following:
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 3]
```diff
@@ -110,3 +110,35 @@ In the storefront, a customer can create a wish list from their account dashboar
...
+
+    - Choose an existing wish list.
+    - Click **Create New Wish List**.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:122]:
```suggestion
    - Chooses an existing wish list.
    - Clicks **Create New Wish List**.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 4]
```diff
@@ -110,3 +110,35 @@ In the storefront, a customer can create a wish list from their account dashboar
...
+
+{:.ee-only}
+## Copy items to a different list
+
+1. Select the checkbox of each item to be moved.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:127]:
```suggestion
1. Selects the checkbox of each item to be moved.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 5]
```diff
@@ -110,3 +110,35 @@ In the storefront, a customer can create a wish list from their account dashboar
...
+
+1. Click **Copy Selected to Wish List** and do one of the following:
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:129]:
```suggestion
1. Clicks **Copy Selected to Wish List** and does one of the following:
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 6]
```diff
@@ -110,3 +110,35 @@ In the storefront, a customer can create a wish list from their account dashboar
...
+
+    - Choose an existing wish list.
+    - Click **Create New Wish List**.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:132]:
```suggestion
    - Chooses an existing wish list.
    - Clicks **Create New Wish List**.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 7]
```diff
@@ -110,3 +110,35 @@ In the storefront, a customer can create a wish list from their account dashboar
...
+
+{:.ee-only}
+## Delete Created Wish Lists
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:135]:
```suggestion
## Delete an existing wish list
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 8]
```diff
@@ -110,3 +110,35 @@ In the storefront, a customer can create a wish list from their account dashboar
...
+
+1. Open the wish list to be deleted.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:137]:
```suggestion
1. Customer opens the wish list to be deleted.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 9]
```diff
@@ -110,3 +110,35 @@ In the storefront, a customer can create a wish list from their account dashboar
...
+
+1. Click the **Delete Wish List** button.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:139]:
```suggestion
1. Clicks <span class="btn">Delete Wish List</span>.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 10]
```diff
@@ -110,3 +110,35 @@ In the storefront, a customer can create a wish list from their account dashboar
...
+
+1. When prompted to confirm, click <span class="btn">OK</span> button.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:141]:
```suggestion
1. When prompted to confirm, clicks <span class="btn">OK</span>.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
For the user guide, the "you" is the merchant who uses the Admin and configures storefront functionality. For procedures that are done by customers on the storefront, we should be writing as third person. There are still many existing items that ned to be corrected, but as we add or edit we are trying to make this changeover.

File: src/customers/account-dashboard-my-wish-list.md (hunk 7 above)
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md]:
```suggestion
## Delete created wish lists
```

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**